torch/__init__.py

```diff
@@ -424,13 +424,36 @@ def use_deterministic_algorithms(d):
         * :func:`torch.bmm`
 
     Note that deterministic operations tend to have worse performance than
-    non-deterministic operations.
+    nondeterministic operations.
```

Comment:
I noticed that sometimes we had "non-deterministic" and sometimes "nondeterministic". Better to be consistent